sympy/series/fourier.py

```diff
@@ -96,19 +96,17 @@ def finite_check(f, x, L):
     def check_fx(exprs, x):
         return x not in exprs.free_symbols
 
-    def check_sincos(expr, x, L):
-        if type(expr) == sin or type(expr) == cos:
-            sincos_args = expr.args[0]
+    def check_sincos(_expr, x, L):
```

Comment:
Is there some specific reason for the change `expr` -> `_expr`

Reply:
I was using Ipython console and found `expr` a part of SymPy. I didn't look much into it but I played safe to avoid any name errors.

Reply:
I usually put the underline at the end in case of name clash as a beginning underline is used to denote a private attribute. (I don't mean that this should be changed.)

Reply:
Yeah I think I still have a lot to learn.

Reply:
`expr` is a module name in SymPy which shouldn't be considered a name clash. `expr` is a common placeholder name to use for expressions.